### PR TITLE
make sure special libraries are not affected by scoring algorithms

### PR DESCRIPTION
**Pull request by yingjieMiao**
@stkem @ahsu1230 

Special library is exempted from complicated scoring algorithm, only if it has not been delivered yet. 
This should fix the persona library reco problem.

### CODE REVIEW

**Comment by yingjieMiao** [comment]
To be more precise, they are still scored, but after final sorting, they will be promoted to the top of the list.

**Comment by ahsu1230** [comment]
thanks!

File: kifi-backend/modules/curator/app/com/keepit/curator/commanders/LibraryRecommendationGenerationCommander.scala [hunk 1]
```diff
@@ -66,10 +66,18 @@ class LibraryRecommendationGenerationCommander @Inject() (
     def scoreReco(reco: LibraryRecommendation) =
       LibraryRecoScore(scoringStrategy.scoreItem(reco.masterScore, reco.allScores, reco.delivered, reco.clicked, None, false, 0f), reco)
 
+    // MAKE DANNY HAPPY. REMOVE THIS
+    def hackySort(scored: Seq[LibraryRecoScore]): Seq[LibraryRecoScore] = {
```
Comment (stkem) [inline kifi-backend/modules/curator/app/com/keepit/curator/commanders/LibraryRecommendationGenerationCommander.scala:70]: My new favorite sorting algorithm :-)

**Comment by stkem** [comment]
:+1:

File: kifi-backend/modules/curator/app/com/keepit/curator/commanders/LibraryRecommendationGenerationCommander.scala (hunk 1 above)
Comment (ahsu1230) [inline kifi-backend/modules/curator/app/com/keepit/curator/commanders/LibraryRecommendationGenerationCommander.scala:70]: we should call it `dannySort()`

**Pull request merged**